### DISCUSSION TITLE
Capturing linear disc motion in a valve using Baram

### DISCUSSION THREAD

**Discussion by jigarmech** [nextfoam/baram#50] [Q&A]
Hi folks, thanks for the wonderful application. Loved it!
I am working on a relief valve simulation for my paper and would like to know if I can capture disc linear motion in the relief valve. I have started using OpenFOAM recently and would like to know if it is possible. The disc is driven by the force beneath it and it opens and closes accordingly provided the mass of the disc.
Is it possible to provide such inputs in the Baram? And where to look if it is.
Thanks so much again!

**Comment by kdg5424** [comment]
Hi, @jigarmech. I'm Donggyu.
First of all, we really thank for your attention and interest in BARAM.

Unfortunately, BARAM can't do what you want.
For doing what you want in BARAM, BARAM shoud work with Dynamic Mesh or 6DOF.
But in now, BARAM doesn't provide Dynamic Mesh or 6DOF method.
We will provide it as soon as possible.
If you want help, please contact below email address.

E-mail : dkkim@nextfoam.co.kr

And fortunately, OpenFOAM works with Dynamic Mesh or 6DOF.

Again, thanks for your attention and interest in BARAM.
